locale/action_names.rb

```diff
@@ -1,5 +1,7 @@
 # Autogenerated!
 _("Action with sub plans")
+_("Check for long running tasks")
+_("Deliver notifications about long running tasks")
```

Comment:
Same question here as in leapp: where do these tasks come from? It's best to update the translations with minimal plugins present.

Reply:
When I did the changes only plugin enabled was foreman_ansible, nothing else

Reply:
https://github.com/theforeman/foreman-tasks/commit/03ce25bcd6f64bee07ffef665db422a89ae09c5c added them to foreman-tasks so we're going to see this in every repo that uses foreman-tasks. No way around it.